PORTING.md

```diff
@@ -0,0 +1,45 @@
+
+
+# Mion porting guild 
+The following document outlines the steps necessary to create a Yocto board support package (BSP)
+
+**NOTE: This process and the files outlined below are subject to change as the format of meta-mion-bsp is codified**
+
+The following platform-specific strings are required for file names and config settings in the BSP:
+- ONL_VENDOR (ONIE_VENDOR) - OEM vendor name as per ONL, using "-" to separate words (the same but with "\_" used as word separator)
+  - e.g. accton, stordis, etc. 
+- ONL_MACHINE (ONIE_MACHINE) - OEM machine name as per ONL, using "-" to separate words (the same but with "\_" used as word separator)
+  - e.g. as4610-30, bf2556c-1t, etc.
+- ONL_ARCH (ONIE_ARCH) - OEM machine arch as per ONL, using "-" as a word separator  (the same but with "\_" used as as word separator)
+  - e.g. x86-64 (x86_64), armel, armhf, etc.
+
+## Create the board support package (BSP) directories and files
+- Check if meta-mion-bsp/meta-mion-<ONL_VENDOR>/ exists and create if necessary 
+  - e.g. meta-mion-bsp/meta-mion-accton/
+- Create platform config in meta-mion-bsp/meta-mion-<ONL_VENDOR>/conf/ based on the existing BSPs. You will need:
+  - conf/layer.conf - similar to existing layer.conf files used for other BSPSvim 
+  - conf/machine/<ONL_MACHINE>.conf - This is a typical Yocto Project machine config. See the Yocto Project documentation on it’s format [link](https://www.yoctoproject.org/docs/current/bsp-guide/bsp-guide.html)
+  - conf/platform-profiles/platform-<OLN_VENDOR>.conf - based on the other platform profiles
+  - Create meta-mion-bsp/meta-mion-<ONL_VENDOR>/conf/onie-profiles/<ONL_VENDOR>-<ONL_MACHINE>/
+  - Populate onlpv{1,2}/{machine,platform}.conf
```

Comment:
Is it worth pointing out this is only possible if there is an existing relevant onlp{1,2} version?